rpy2/robjects/help.py

```diff
@@ -262,10 +262,15 @@ def usage(self) -> str:
         """ Get the usage for the object """
         return self._get_section(r'\usage')
 
-    def iteritems(self):
+    def items(self):
         """ iterator through the sections names and content
         in the documentation Page. """
-        return self.sections.iteritems
+        return self.sections.items()
+
+    def iteritems(self):
+        """ iterator through the sections names and content
+        in the documentation Page. (deprecated, use items()) """
```

Comment:
In Python, there is mechanism to make deprecation more visible than only a mention in the docstring. I am adding it.